modules/data-entry/src/main/frontend/src/dataHomepage/EditButton.jsx

```diff
@@ -31,7 +31,7 @@ import QuestionnaireStyle from "../questionnaire/QuestionnaireStyle.jsx";
 function EditButton(props) {
   const { entryPath, entryType, buttonClass, admin } = props;
   return(
-    <Link to={(admin ? "/content.html/admin" : "/content.html") + entryPath}>
+    <Link to={(admin ? "/content.html/admin" : "/content.html") + entryPath + (entryType == "Form" ? ".edit" : "")}>
```

Comment:
I think we can use `.edit` for all node types, not just forms. Right now there's no other table using it, having it in the URL doesn't seem to break anything, and we expect all edits to go through this URL in the future.

```suggestion
    <Link to={(admin ? "/content.html/admin" : "/content.html") + entryPath + ".edit"}>
```